node/src/actors/chain_manager/mod.rs

```diff
@@ -1029,8 +1029,113 @@ impl ChainManager {
 
                     for co_tx in &block.txns.commit_txns {
                         let commit_pkh = co_tx.body.proof.proof.pkh();
+                        let dr_output = self
+                            .chain_state
+                            .data_request_pool
+                            .get_dr_output(&co_tx.body.dr_pointer)
+                            .unwrap();
+                        let collateral = dr_output.collateral;
+                        let commit_fee = dr_output.commit_and_reveal_fee;
+                        // Reset witnessing power
                         let _ =
                             stakes.reset_age(commit_pkh, Capability::Witnessing, current_epoch, 1);
+                        // Subtract collateral from staked balance
+                        let _ = stakes.reserve_collateral(commit_pkh, Wit::from(collateral));
+                        // Add commit reward
+                        let _ = stakes.add_reward(
+                            commit_pkh,
```

Comment:
Wouldn't `dr_output.commit_and_reveal_fee` rather be sent to `miner_pkh`?

Reply:
![youre-god-damn-right](https://github.com/user-attachments/assets/fd22e65b-f963-4d28-8706-0aef308ba46a)